iceoryx_examples/icedelivery/iox_publisher.cpp

```diff
@@ -45,65 +47,85 @@ int main()
 
     iox::runtime::PoshRuntime::initRuntime(APP_NAME);
 
+    //! [create publisher]
     iox::popo::Publisher<RadarObject> publisher({"Radar", "FrontLeft", "Object"});
+    //! [create publisher]
 
     double ct = 0.0;
     while (!killswitch)
     {
         ++ct;
+        double sampleValue1 = ct + 89;
+        double sampleValue2 = ct + 144;
+        double sampleValue3 = ct + 233;
+        double sampleValue4 = ct + 377;
 
         // API Usage #1
         //  * Retrieve a typed sample from shared memory.
         //  * Sample can be held until ready to publish.
         //  * Data is default constructed during loan
+        //! [usage1]
         publisher.loan()
```

Comment:
have you thought about including the comments? It's not directly described in the README so it might help to have it inlcuded.